test/DynamoCoreTests/DSLibraryTest.cs

```diff
@@ -121,6 +121,32 @@ public void TestLoadDllFileSuccess()
             Assert.IsTrue(LibraryLoaded);
         }
 
+        [Test]
+        [Category("UnitTests")]
+        public void TestLoadDllFileAtPathSuccess()
+        {
+            LibraryLoaded = false;
+
+            string libraryPath = Path.Combine(TestDirectory, "pkgs", "Package", "bin", @"Package.dll");
+        
+            libraryServices.ImportLibrary(libraryPath);
+   
+            Assert.IsTrue(LibraryLoaded);
+        }
+
+        [Test]
+        [Category("UnitTests")]
+        public void TestLoadDllFileWithApostropheInPathSuccess()
+        {
+            LibraryLoaded = false;
+
+            string libraryPath = Path.Combine(TestDirectory, "pkgs", "Pack'age", "bin", @"Package.dll");
```

Comment:
@QilongTang This test fails with out the formatting fix

Reply:
@saintentropy maybe I misunderstand the issue, how would `fileName.Replace("\\'", "'")` help this case?

Reply:
Sorry now I get your question @QilongTang.  The path here gets modified here

https://github.com/DynamoDS/Dynamo/blob/master/src/Engine/ProtoCore/Utils/CompilerUtils.cs#L258 in `TryLoadAssemblyIntoCore`

The return value of `ToLiteral` is modified from _Pack'age_ to _Pack\\'age_ when it is parsed here https://github.com/DynamoDS/Dynamo/blob/master/src/Engine/ProtoCore/Utils/CompilerUtils.cs#L242

Reply:
![image](https://user-images.githubusercontent.com/2145751/82509293-792e2400-9ad5-11ea-8afe-b885877d8d23.png)

Reply:
Above is one example of the of the call to `ToLiteral`